app/content-panel/blogs/[id]/page.tsx

```diff
@@ -11,7 +11,7 @@ const BlogDetailsPage = ({ params }: { params: { id: string } }) => {
   const [slug, setSlug] = useState('N/A');
   const [image, setImage] = useState('');
   const [author, setAuthor] = useState({ first_name: 'N/A', last_name: 'N/A' });
-  const [publish_date, setPublishDate] = useState('N/A');
+  const [publishDate, setPublishDate] = useState('N/A');
```

Comment:
all this fields/states will have the state for that blogpost we have fetched from the api
Here we are viewing and later editing and deleting existing blog posts.
For a blog post to be created, most of this fields would be required.